runtime/t0rn-parachain/src/circuit_config.rs

```diff
@@ -125,6 +126,68 @@ impl pallet_clock::traits::OnHookQueues<Runtime> for GlobalOnInitQueues {
     }
 }
 
+parameter_types! {
+    // TODO: update me to be better
+    pub const EscrowAccount: AccountId = AccountId::new([51_u8; 32]);
+    pub const RewardMultiplier: Balance = 1;
+    pub const MinNominatorBond: Balance = 1;
+    pub const MinAttesterBond: Balance = 1;
+    pub const DefaultCommission: Percent = Percent::from_percent(10);
+}
+
+impl pallet_attesters::Config for Runtime {
+    type ActiveSetSize = ConstU32<32>;
+    type BatchingWindow = ConstU32<6>;
+    type CommitmentRewardSource = EscrowAccount;
+    type CommitteeSize = ConstU32<16>;
+    type Currency = Balances;
+    type DefaultCommission = DefaultCommission;
+    type Event = Event;
+    type MaxBatchSize = ConstU32<128>;
+    type MinAttesterBond = MinAttesterBond;
+    type MinNominatorBond = MinNominatorBond;
+    type RandomnessSource = RandomnessCollectiveFlip;
+    type RewardMultiplier = RewardMultiplier;
+    type ShufflingFrequency = ConstU32<400>;
+    type SlashAccount = EscrowAccount;
+}
+
+use t3rn_primitives::monetary::TRN;
+
+parameter_types! {
+    pub const TotalInflation: Perbill = Perbill::from_parts(4_400_0000); // 4.4%
+    pub const AttesterInflation: Perbill = Perbill::from_parts(1_100_0000); // 1.1%
+    pub const ExecutorInflation: Perbill = Perbill::from_parts(0_800_0000); // 0.8%
+    pub const CollatorInflation: Perbill = Perbill::from_parts(0_500_0000); // 0.5%
+    pub const TreasuryInflation: Perbill = Perbill::from_parts(2_000_0000); // 2%
```

Comment:
Why inflate the treasury? Thought treasury funds would simply be allocated at genesis.

Reply:
2% per year for parachain treasury as per https://docs.t3rn.io/about_t3rn/token-economics/tokenomics-inflation